app/Domain/Integrations/Controllers/IntegrationController.php

```diff
@@ -257,29 +279,9 @@ public function activateWithOrganization(string $id, CreateOrganizationRequest $
         );
     }
 
-    public function show(string $id): Response
-    {
-        try {
-            $integration = $this->integrationRepository->getById(Uuid::fromString($id));
-            $subscription = $this->subscriptionRepository->getById($integration->subscriptionId);
-            $contacts = $this->contactRepository->getByIntegrationId(UUid::fromString($id));
-        } catch (Throwable) {
-            abort(404);
-        }
-
-        return Inertia::render('Integrations/Detail', [
-            'integration' => [
-                ...$integration->toArray(),
-                'contacts' => $contacts->toArray(),
-                'urls' => $integration->urls(),
-                'organization' => $integration->organization(),
-                'subscription' => $subscription,
-            ],
-        ]);
-    }
-
-    public function showWidget(IntegrationModel $integration): RedirectResponse
+    public function showWidget(string $id): RedirectResponse
```

Comment:
I was able to use the model binding, even inside a test, after running `php artisan optimize`

Reply:
I couldn't make it work. I'm leaving it as is for now.